test/lib/InputProcessor.test.ts

```diff
@@ -129,34 +129,16 @@ describe("InputProcessorImpl Tests", async () => {
 			const inputs: Inputs = {
 				target: ['thisFileDoesNotExist.xml', 'thisFileAlsoDoesNotExist.json']
 			};
-			let errorThrown: boolean;
-			let message: string;
-			try {
-				inputProcessor.resolveProjectDirPaths(inputs);
-				errorThrown = false;
-			} catch (e) {
-				errorThrown = true;
-				message = e.message;
-			}
-			expect(errorThrown).to.equal(true, 'Error should have been thrown');
-			expect(message).to.equal(getMessage(BundleName.CommonRun, 'validations.noFilesFoundInTarget'));
+			const projectDirPaths: string[] = inputProcessor.resolveProjectDirPaths(inputs);
+			expect(projectDirPaths).to.deep.equal([process.cwd()]);
 		})
 
 		it("Unspecified projectdir with glob target that resolves to no files", async () => {
```

Comment:
@stephen-carter-at-sf - Can we add one test where inputs is empty?

Reply:
Yes. Coming soon.